akka-javasdk/src/main/scala/akka/javasdk/impl/eventsourcedentity/EventSourcedEntityImpl.scala

```diff
@@ -127,7 +121,7 @@ private[impl] final class EventSourcedEntityImpl[S, E, ES <: EventSourcedEntity[
           // FIXME smuggling 0 arity method called from component client through here
           ScalaPbAny.defaultInstance.withTypeUrl(AnySupport.JsonTypeUrlPrefix).withValue(ByteString.empty())))
     val metadata: Metadata =
-      MetadataImpl.of(Nil) // FIXME MetadataImpl.of(command.metadata.map(_.entries.toVector).getOrElse(Nil))
+      MetadataImpl.Empty // FIXME MetadataImpl.of(command.metadata.map(_.entries.toVector).getOrElse(Nil))
```

Comment:
You can use `MetadataImpl.of(command.metadata)` now.

Reply:
ah, maybe I have to here, because a test is failing:
```
expected: <test-topic-metadata> but was: <test-topic>
```